src/main/java/frc/robot/Constants.java

```diff
@@ -36,6 +32,24 @@
  */
 public final class Constants {
     public static final class RobotMap{
+         // PID constants
+
+     public static double kPanelP = 0.1;
```

Comment:
Probably want to specify these are placeholders since these are the gains copied from Rev's example github